### PR TITLE
Release notes 2.13.4

### PR DESCRIPTION
**Pull request by znicholasbrown**

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Prefect Release Notes
 
+## Release 2.13.4
+
+### Fixes
+
+- Fixes an issue with infinite scrolling on the sub flow runs tab in the UI - https://github.com/PrefectHQ/prefect-ui-library/pull/1788
+
+### Documentation
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:10]: - Drop beta tags from push work pools documentation — https://github.com/PrefectHQ/prefect/pull/10799

Comment (serinamarie) [inline RELEASE-NOTES.md:10]:
```suggestion
- Drop beta tags from push work pools documentation — https://github.com/PrefectHQ/prefect/pull/10799
```

Comment (serinamarie) [inline RELEASE-NOTES.md:10]:
```suggestion
- Add dark mode base job template screenshot to work pools documentation — https://github.com/PrefectHQ/prefect/pull/10849
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # Prefect Release Notes
 
+## Release 2.13.4
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:4]:
```suggestion
### Enhancements
- Allow task runs without a flow run — https://github.com/PrefectHQ/prefect/pull/10816
```

Comment (serinamarie) [reply RELEASE-NOTES.md:4]: There's a better way to say this

Comment (znicholasbrown) [reply RELEASE-NOTES.md:4]: I'm not totally sure this is worth calling out yet tbh - support for this is technically possible but not exactly friendly

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+### Fixes
+
+- Fixes an issue with infinite scrolling on the sub flow runs tab in the UI - https://github.com/PrefectHQ/prefect-ui-library/pull/1788
+
+### Documentation
```
Comment (znicholasbrown) [inline RELEASE-NOTES.md:9]:
```suggestion
### Documentation

```

File: RELEASE-NOTES.md (hunk 2 above)
Comment (serinamarie) [reply RELEASE-NOTES.md:4]: It would feel strange to me not to include this

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,18 @@
 # Prefect Release Notes
 
+## Release 2.13.4
+### Enhancements
+- Allow task runs without a flow run — https://github.com/PrefectHQ/prefect/pull/10816
```
Comment (znicholasbrown) [inline RELEASE-NOTES.md:5]:
```suggestion

- Allow task runs without a flow run — https://github.com/PrefectHQ/prefect/pull/10816

```

File: RELEASE-NOTES.md (hunk 2 above)
Comment (serinamarie) [reply RELEASE-NOTES.md:7]: if users look to the release notes to upgrade, it makes sense for any possibly breaking changes to be there

Comment (serinamarie) [reply RELEASE-NOTES.md:7]: we could put it under the ###experimental heading..

Comment (serinamarie) [reply RELEASE-NOTES.md:7]: how about "Lay groundwork for allowing task runs without a flow run"?

Comment (znicholasbrown) [reply RELEASE-NOTES.md:7]: I don't think it warrants that - it's fine under the enhancements section

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,20 @@
 # Prefect Release Notes
 
+## Release 2.13.4
+### Enhancements
+
+- Allow task runs without a flow run — https://github.com/PrefectHQ/prefect/pull/10816
+
+### Fixes
+
+- Fixes an issue with infinite scrolling on the sub flow runs tab in the UI - https://github.com/PrefectHQ/prefect-ui-library/pull/1788
```
Comment (serinamarie) [inline RELEASE-NOTES.md:10]:
```suggestion
- Fix an issue with infinite scrolling on the sub flow runs tab in the UI - https://github.com/PrefectHQ/prefect-ui-library/pull/1788
```

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,20 @@
 # Prefect Release Notes
 
+## Release 2.13.4
+### Enhancements
+
+- Allow task runs without a flow run — https://github.com/PrefectHQ/prefect/pull/10816
```
Comment (serinamarie) [inline RELEASE-NOTES.md:6]: I think something that emphasizes that it's not really user-friendly yet, such as the phrase "laying groundwork" or equivalent, might help

**Review by serinamarie — APPROVED** [review]
Thanks!

File: RELEASE-NOTES.md (hunk 6 above)
Comment (serinamarie) [reply RELEASE-NOTES.md]: awesome, thanks!

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,20 @@
 # Prefect Release Notes
 
+## Release 2.13.4
+### Enhancements
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:5]:
```suggestion
```

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,20 @@
 # Prefect Release Notes
 
+## Release 2.13.4
```
Comment (serinamarie) [inline RELEASE-NOTES.md:3]:
```suggestion

## Release 2.13.4
```

File: RELEASE-NOTES.md [hunk 9]
```diff
@@ -1,5 +1,20 @@
...
+### Enhancements
+
+- Lift API and database constraints that require task runs to have an associated flow run id — https://github.com/PrefectHQ/prefect/pull/10816
+
+### Fixes
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:10]:
```suggestion
```

File: RELEASE-NOTES.md [hunk 10]
```diff
@@ -34,6 +49,9 @@ You can now set your own default filter view on your Flow Runs page! You must fi
 
 Note: The previous default view ("Default view") has been renamed to "Past week".
 
+## Integrations
+New Google Vertex AI work pool and worker. Run flows in containers on Google Vertex AI. Requires a Google Cloud Platform account and prefect-gcp installed. Read more [here](https://prefecthq.github.io/prefect-gcp/vertex_worker/).
```
Comment (serinamarie) [inline RELEASE-NOTES.md:52]: @discdiver  can you make this a bullet point?

File: RELEASE-NOTES.md [hunk 11]
```diff
@@ -1,5 +1,19 @@
...
+## Release 2.13.4
+
+### Enhancements
+- Lift API and database constraints that require task runs to have an associated flow run id — https://github.com/PrefectHQ/prefect/pull/10816
+
+### Fixes
+- Fix an issue with infinite scrolling on the sub flow runs tab in the UI - https://github.com/PrefectHQ/prefect-ui-library/pull/1788
+
+### Documentation
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:12]:
```suggestion
```

File: RELEASE-NOTES.md [hunk 12]
```diff
@@ -34,6 +48,9 @@ You can now set your own default filter view on your Flow Runs page! You must fi
 
 Note: The previous default view ("Default view") has been renamed to "Past week".
 
+## Integrations
+New Google Vertex AI work pool and worker. Run flows in containers on Google Vertex AI. Requires a Google Cloud Platform account and prefect-gcp installed. Read more [here](https://prefecthq.github.io/prefect-gcp/vertex_worker/).
```
Comment (znicholasbrown) [inline RELEASE-NOTES.md:52]:
```suggestion

- New Google Vertex AI work pool and worker. Run flows in containers on Google Vertex AI. Requires a Google Cloud Platform account and prefect-gcp installed. Read more [here](https://prefecthq.github.io/prefect-gcp/vertex_worker/).
```

Comment (serinamarie) [inline RELEASE-NOTES.md:52]:
```suggestion
- Add Google Vertex AI work pool and worker - https://github.com/PrefectHQ/prefect-gcp/pull/211
```

**Review by discdiver — APPROVED** [review]

File: RELEASE-NOTES.md [hunk 13]
```diff
@@ -34,6 +47,12 @@ You can now set your own default filter view on your Flow Runs page! You must fi
 
 Note: The previous default view ("Default view") has been renamed to "Past week".
 
+## Integrations
+
+New Google Vertex AI work pool and worker.
```
Comment (serinamarie) [inline RELEASE-NOTES.md:52]:
```suggestion
## New Google Vertex AI work pool and worker
```

**Review by serinamarie — APPROVED** [review]

**Pull request merged**